rigs/heft-web-rig/profiles/library/config/rush-project.json

```diff
@@ -1,3 +1,10 @@
 {
-  "projectOutputFolderNames": ["dist", "lib", "lib-commonjs", "temp"]
+  "projectOutputFolderNames": ["dist", "lib", "lib-commonjs", "temp"],
```

Comment:
Why does this appear in two places?

Reply:
This one is for non-phased builds. The other is the list of folders cached by the `_phase:build` phase.